README.md

```diff
@@ -91,6 +98,23 @@ else fails, try removing the virtual environment and reinstalling:
 
 [local5000]:http://localhost:5000/
 
+### Troubleshooting Possible Errors
+
+* Should you get series of type errors that looks something like `npm ERR! typeerror Error: Missing required argument #1`, after running `pipenv run lektor server -f webpack`, this is most likely due to running an older version of Node.js.
+
+  As stated earlier in the prerequisites, you should be running Node.js versions 12+.
+
+  Follow [this](https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions) tutorial to upgrade your node version (for GNU/Unix systems).
+
+* Should you get an `OSError: [Errno 28] inotify watch limit reached` after running any command, this means that your system file watcher is running out of alloted handles, usually because the workspace is large and contains many files. 
+
+  The solution is to run:
+
+  `sudo sysctl fs.inotify.max_user_watches=524288`
+
+  This increases your `inotify` watch limit (for the session) to 524288, which is the maximum value and is also enough to allow the  setup go through.
+
+  You can learn more about file watchers [from this blog post](https://unixia.wordpress.com/2018/04/28/inotify-watch-limit-reached-wait-what/) or [from the VS Code documentation](https://code.visualstudio.com/docs/setup/linux#_visual-studio-code-is-unable-to-watch-for-file-changes-in-this-large-workspace-error-enospc).
```

Comment:
You can put bundle:
- Lines 111 and 113 together.
- Lines 115 and 117 together.